KnockKnock-iOS/Scenes/Cells/FeedListCell.swift

```diff
@@ -0,0 +1,302 @@
+    $0.backgroundColor = .white
+    $0.clipsToBounds = true
+    $0.layer.cornerRadius = 5
+    $0.layer.masksToBounds = false
+    $0.layer.shadowColor = UIColor.black.cgColor
+    $0.layer.shadowOpacity = 0.1
+    $0.layer.shadowOffset = CGSize(width: 0, height: 10)
+    $0.layer.shadowRadius = 5
+  }
+
+  let contentLabel = UILabel().then {
+    $0.translatesAutoresizingMaskIntoConstraints = false
+    $0.numberOfLines = 0
+    $0.backgroundColor = .white
+    $0.font = UIFont(name: "Apple SD Gothic Neo", size: 13)
+    $0.text = "패키지 상품을 받았을때의 기쁨 후엔 늘 골치아픈 쓰레기와 분리수거의 노동시간이 뒤따릅니다. 패키지 상품을 받았을때의 기쁨 후엔 늘 골치아픈 쓰레기와 분리수거의 노동시간이 뒤따릅니다. "
+  }
+
+  let likeButton = UIButton().then {
+    $0.translatesAutoresizingMaskIntoConstraints = false
+    $0.setImage(KKDS.Image.ic_like_24_off, for: .normal)
+    $0.setTitle(" 0", for: .normal)
+    $0.contentHorizontalAlignment = .center
+    $0.semanticContentAttribute = .forceLeftToRight
+  }
+
+  let commentsButton = UIButton().then {
+    $0.translatesAutoresizingMaskIntoConstraints = false
+    $0.setImage(KKDS.Image.ic_balloon_24_gr, for: .normal)
+    $0.setTitle(" 0", for: .normal)
+    $0.contentHorizontalAlignment = .center
+    $0.semanticContentAttribute = .forceLeftToRight
+  }
+
+  override func prepareForReuse() {
+    super.prepareForReuse()
+    self.imageScrollView.subviews.forEach{
+      $0.removeFromSuperview()
+    }
+    self.imageNumberLabel.isHidden = true
+    self.imagePageControl.isHidden = true
+  }
+
+  // MARK: - Bind
+
+  func bind(feed: Feed) {
+    self.userIdLabel.text = "\(feed.userId)"
+    self.contentLabel.text = feed.content
+    self.setImageView(images: feed.images, scale: feed.scale)
+
+    if feed.images.count > 1 {
+      self.imageNumberLabel.isHidden = false
+      self.imagePageControl.isHidden = false
+      self.imagePageControl.numberOfPages = feed.images.count
+      self.imageNumberLabel.text = "1/\(feed.images.count)"
+    }
+    
+    guard let contentTextLength = self.contentLabel.text?.count
+    else {
+      return
+    }
+
+    if contentTextLength > 1 {
+      DispatchQueue.main.async {
+        self.contentLabel.addTrailing(with: "... ", moreText: "더보기", moreTextFont: .systemFont(ofSize: 13), moreTextColor: UIColor.gray)
+      }
+    }
+  }
+
+  func setImageView(images: [String], scale: String) {
+    for index in 0..<images.count {
+
+      let imageView = UIImageView()
+      imageView.image = UIImage(named: images[index])
+      imageView.contentMode = .scaleAspectFill
+      imageView.layer.cornerRadius = 5
+      imageView.clipsToBounds = true
+
+      let xPosition = self.contentView.frame.width * CGFloat(index)
+      let width = self.contentView.frame.width
+
+      switch scale {
+      case "3:4":
+        imageView.frame = CGRect(x: xPosition,
+                                 y: 0,
+                                 width: width,
+                                 height: width * 1.333)
```

Comment:
```suggestion
        imageView.frame = CGRect(
          x: xPosition,
          y: 0,
          width: width,
          height: width * 1.333
        )
```
저희 컨벤션에 의하면 개행은 이런 방식으로 👍